packages/react-cosmos-xhr-proxy/src/__tests__/mocking.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import createXhrProxy from '../';
+
+// Vars populated in beforeEach blocks
+let XhrProxy;
+let NextProxy;
+let nextProxyNext;
+let nextProxy;
+let Component;
+let componentRef;
+let onComponentRef;
+let onFixtureUpdate;
+let wrapper;
+let props;
+
+beforeEach(() => {
+  XhrProxy = createXhrProxy();
+
+  // Objects to check identity against
+  NextProxy = () => {};
+  nextProxyNext = {};
+  nextProxy = {
+    value: NextProxy,
+    next: () => nextProxyNext,
+  };
+  Component = () => {};
+  componentRef = {};
+  onComponentRef = jest.fn();
+  onFixtureUpdate = jest.fn();
+
+  return new Promise(resolve => {
+    wrapper = shallow(
+      <XhrProxy
+        nextProxy={nextProxy}
+        component={Component}
+        fixture={{
+          foo: 'bar',
+          xhr: [
+            {
+              url: '/users',
+              response: (req, res) =>
+                res.status(200).body([{ name: 'John' }, { name: 'Jessica' }]),
+            },
+            {
+              url: '/user',
```

Comment:
Just `/user` I think. It just passes it to [xhr-mock](https://github.com/jameslnewell/xhr-mock#api). 

fetch-mock seems more flexible in this regard:
![image](https://user-images.githubusercontent.com/250750/28753705-c96bb760-7540-11e7-8498-818d3655cf71.png)

Ideally both xhr and fetch requests would be mocked in the same way, but merging the APIs would've been risky business so I went for the simple approach–just use their own API and avoid maintenance costs.